content_sync/pipelines/concourse.py

```diff
@@ -249,3 +259,98 @@ def get_build_status(self, build_id: int):
     def abort_build(self, build_id: int):
         """Abort a build"""
         return self.api.abort_build(build_id)
+
+
+class ThemeAssetsPipeline(BaseThemeAssetsPipeline):
+    """
+    Concourse-CI pipeline for publishing theme assets
+    """
+
+    MANDATORY_SETTINGS = [
```

Comment:
I moved these in my PR so they could be used for all concourse pipelines: https://github.com/mitodl/ocw-studio/pull/950/files#diff-979f137beba738446b3fc46c89be033a4f04e2e285ce7e68d3028b0e1b83c11fR23

Reply:
The thing is, the theme pipeline doesn't require the Github settings.  I don't know if you saw, but I added a utility function called `check_mandatory_settings` to utils.py to abstract the actual check, then each pipeline can define what settings are mandatory.  I think in `concourse.py` at least, it makes sense for each pipeline to include the `CONCOURSE_` prefixed settings, then define what they will use beyond that?

Reply:
ok, makes sense